src/theme.js

```diff
@@ -12,6 +12,9 @@ export const color = {
   spaceLighterAlpha: 'rgba(0,19,25,0.2)',
   spaceLightestAlpha: 'rgba(0,19,25,0.1)',
 
+  nova: 'rgb(255, 255, 255)',
+  novaAlpha: 'rgba(255,255,255,0.8)',
```

Comment:
These are all the colors that are defined in the design system?

Reply:
I'll ask Jelle.